index.html

```diff
@@ -373,6 +373,29 @@ <h2>
             NO
           </td>
         </tr>
+        <tr data-cite="window-placement">
+          <td class="string-token">
+            "[=window-management=]"
```

Comment:
From the permission side of things https://github.com/w3c/window-placement/blob/main/index.bs#L732 looks good.